sys/cloudformation/stack.yaml

```diff
@@ -97,10 +100,18 @@ Parameters:
     ConstraintDescription: >
       VPC Id must begin with 'vpc-' or leave blank to have a new VPC created
 
+  EnvAPPDEBUG:
+    Type: String
+    Default: '0'
+    Description: Environment variable for APP_DEBUG
   EnvAPPENV:
     Type: String
     Default: 'prod'
     Description: Environment variable for APP_ENV
+  EnvAPPENVCanary:
+    Type: String
+    Default: 'canary'
+    Description: Environment variable for APP_ENV (dark-canary)
```

Comment:
It's not clear to me what is the intent of this variable, neither what are the expected values .. could you argument in the description? :*